source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Why I deleted my LinkedIn account"
+perex: '''
+    I used LinkedIn since 2008 when it was sending spam to all my friends. After 8 years of tuning my profile to 95 % score, reading mostly copy-pasted messages,
+    checking false notifications, flirting with HR girls and hoping for up votes for my recent skills, <strong>I've decided it is time to let it go</strong>. Why and what I'm gonna do now?
+'''
+lang: en
+---
+
+*Disclaimer: even if I get few abroad offers, my experience is related to work environment in the Czech Republic.*
+
+I was reviewing my 2016 on social networks disconnection: I reduced 350 friends on Facebook to 20, unfollowed all but 2 groups and I dropped all people I follow on Twitter. 
+My attention got better and I could focus on things that really mattered to me. That's why I started though about the next step.
```

Comment:
Maybe... My attention got better and I could focus on things that really matter to me. That's why I started thinking about the next step.

And one more thing: You said you "started thinking about the next step" because of you "could focus on things that really matter". Is that really the main cause?